aio_pika/abc.py

```diff
@@ -692,10 +694,29 @@ async def close(self, exc: Optional[aiormq.abc.ExceptionType]) -> Any:
             await self.close_callback.wait()
 
 
+@dataclass
+class ConnectionParameter:
+    name: str
+    parser: Callable[[str], Any]
+    default: Optional[str] = None
+    kwarg: bool = False
```

Comment:
I think it would be better to use "is_" prefix with boolean variable names: "is_kwarg".